src/cata_utility.cpp

```diff
@@ -193,10 +194,16 @@ double logarithmic_range( int min, int max, int pos )
     const double LOGI_MIN = logarithmic( -LOGI_CUTOFF );
     const double LOGI_MAX = logarithmic( +LOGI_CUTOFF );
     const double LOGI_RANGE = LOGI_MAX - LOGI_MIN;
-    // Anything beyond [min,max] gets clamped.
-    if( pos < min ) {
+
+    if( min >= max ) {
+        debugmsg( "Invalid range" );
```

Comment:
> That debugmsg could have more details

You're right. I thought that macro debugmsg() does most of that and it actually does. It writes: file name, line number and no function name (weird) into a log file only. I found, that some files use DebugLog() along with debugmsg() to report an error, others don't. What's the most appropriate way of reporting and logging errors you'd use?

> Also, requiring min to be strictly lesser than max (not allowing equality) could break something. Morale code uses it.

It won't break anything for now, I've taken it into account (see the first comment).

> equal limits could have been sent by player::add_morale() regularly, but luckily expired morale_points are removed.